help/actions/_posts/2015-02-09-export.md

```diff
@@ -74,27 +74,35 @@ The same format as above but encoded for use as a rich text document (rtf) which
+| text                                                                                                                           	| in full if available 	| same                     	| lastname, firstname m.                                           	| **%A**    	| , and xxx colleagues 	|  -Henneken, E. A., Muench, G., Holm Nielsen, L., Blanco-Cuaresma, S. and Accomazzi, A.<br>-Kurtz, Michael J. and Accomazzi, Alberto 	|  -Henneken, E. A., Muench, G., and 4 colleagues<br>-Kurtz, Michael J. and Accomazzi, Alberto 	|
+| symbol                                                                                                                         	| in full if available 	| same                     	| lastname, firstname m.                                           	| **%a**    	| , et al.             	|  -Henneken, E. A., Muench, G., Holm Nielsen, L., Blanco-Cuaresma, S., & Accomazzi, A.<br>-Kurtz, Michael J. & Accomazzi, Alberto    	|  -Henneken, E. A., Muench, G., et al.<br>-Kurtz, Michael J. & Accomazzi, Alberto             	|
+| none                                                                                                                           	| initials             	| same                     	| lastname, f. m.                                                  	| **%G**    	| , et al.             	|  -Henneken, E. A., Muench, G., Holm Nielsen, L., Blanco-Cuaresma, S., Accomazzi, A.<br>-Kurtz, M. J., Accomazzi, A.                 	|  -Henneken, E. A., Muench, G., et al.<br>-Kurtz, M. J., Accomazzi, A.                        	|
+| none                                                                                                                           	| initials             	| same                     	| lastname f. m.<br><br>note no comma after lastname                       	| **%g**    	| , et al.             	|  -Henneken E. A., Muench G., Holm Nielsen L., Blanco-Cuaresma S., Accomazzi A.<br>-Kurtz M. J., Accomazzi A.                        	|  -Henneken E. A., Muench G., et al.<br>-Kurtz M. J., Accomazzi A.                            	|
+| text                                                                                                                           	| initials             	| different                	| first author: lastname, f. m.<br><br>other authors: f. m., lastname      	| **%I**    	| , and xxx colleagues 	|  -Henneken, E. A., G. Muench, L. Holm Nielsen, S. Blanco-Cuaresma, and A. Accomazzi<br>-Kurtz, M. J. and A. Accomazzi               	|  -Henneken, E. A., G. Muench, and 4 colleagues<br>-Kurtz, M. J. and A. Accomazzi             	|
```

Comment:
* for both this one and %i, there's a typo in the format column: "other authors: f. m., lastname" shouldn't have a comma (unless there's an error in the example column instead?), so should be "other authors: f. m. lastname" instead